airsonic-main/src/main/java/org/airsonic/player/controller/PodcastChannelController.java

```diff
@@ -53,13 +53,14 @@ protected ModelAndView handleRequestInternal(HttpServletRequest request, HttpSer
 
         Map<String, Object> map = new HashMap<>();
         ModelAndView result = new ModelAndView();
-        result.addObject("model", map);
 
         int channelId = ServletRequestUtils.getRequiredIntParameter(request, "id");
 
         map.put("user", securityService.getCurrentUser(request));
         map.put("channel", podcastService.getChannel(channelId));
         map.put("episodes", podcastService.getEpisodes(channelId));
+
+        result.addObject("model", map);
```

Comment:
change not needed?

Reply:
Absolutely unnecessary ;-) Going to revert that...